docs/spec.md

```diff
@@ -2805,6 +2805,7 @@ indices and explains in detail which `operand` indices they correspond to.
 
```

Comment:
Also, what do we do with the image? Do we need to update it as well?

Reply:
The svg export did its magic, so I couldn't quite make the changes as minimal as hoped.